src/app/payment/components/checkout/ProductCardComponent.tsx

```diff
@@ -1,65 +1,87 @@
+import TextInput from '../../../share/components/ShareItemDialog/components/TextInput';
+import Button from '../../../shared/components/Button/Button';
+import { useThemeContext } from '../../../theme/ThemeProvider';
 import { ReactComponent as GuaranteeDarkDays } from 'assets/icons/checkout/guarantee-dark.svg';
 import { ReactComponent as GuaranteeWhiteDays } from 'assets/icons/checkout/guarantee-white.svg';
-import { useState } from 'react';
-import { CouponCodeData, Currency, CurrentPlanSelected } from '../../types';
+import { CouponCodeData, Currency, RequestedPlanData } from '../../types';
 
 interface ProductFeaturesComponentProps {
-  selectedPlan: CurrentPlanSelected;
-  couponCodeData?: CouponCodeData;
-  couponError?: string;
+  selectedPlan: RequestedPlanData;
+  users: number;
   upsellManager: UpsellManagerProps;
+  onUsersChange: (users: number) => void;
   onRemoveAppliedCouponCode: () => void;
   onCouponInputChange: (promoCode: string) => void;
+  couponCodeData?: CouponCodeData;
+  couponError?: string;
 }
 
 const Separator = () => <div className="border border-gray-10" />;
 
-export const getProductAmount = (amount: DisplayPrice['amount'], couponCodeData?: CouponCodeData): number => {
+export const getProductAmount = (
+  amount: DisplayPrice['amount'],
+  users: number,
+  couponCodeData?: CouponCodeData,
+): number => {
   if (couponCodeData?.amountOff) {
-    return amount - couponCodeData.amountOff / 100;
+    return (amount - couponCodeData.amountOff / 100) * users;
   }
 
   if (couponCodeData?.percentOff) {
     const discount = 100 - couponCodeData.percentOff;
 
-    return (amount * discount) / 100;
+    return ((amount * discount) / 100) * users;
   }
 
-  return amount;
+  return amount * users;
 };
 
 export const ProductFeaturesComponent = ({
   selectedPlan,
   couponCodeData,
   couponError,
+  users,
   upsellManager,
+  onUsersChange,
   onRemoveAppliedCouponCode,
   onCouponInputChange,
 }: ProductFeaturesComponentProps) => {
   const { translate, translateList } = useTranslationContext();
   const { checkoutTheme } = useThemeContext();
-
   const [couponName, setCouponName] = useState<string>('');
-  const { isUpsellSwitchActivated, showUpsellSwitch, onUpsellSwitchButtonClicked } = upsellManager;
-
   const bytes = bytesToString(selectedPlan.bytes);
-  const features = translateList('checkout.productCard.planDetails.features', {
-    spaceToUpgrade: bytes,
-  });
 
+  const { isUpsellSwitchActivated, showUpsellSwitch, onUpsellSwitchButtonClicked } = upsellManager;
+  const isBusiness = selectedPlan.type === UserType.Business;
+  const perUserLabel = isBusiness ? translate('checkout.productCard.perUser') : undefined;
+  const totalLabel = isBusiness
+    ? translate('checkout.productCard.totalForBusiness', {
+        N: users,
+      })
+    : translate('checkout.productCard.total');
+  const maxUploadGBfile = bytes === '1TB' ? '5GB' : '20GB';
```

Comment:
In this case, each business plan has a limit for uploading files, for the standard plan 5GB and the pro 20GB.

Instead of duplicating the features for each business plan, I manage some values that change in the text from the components, and this is one of them. Maybe I can handle this data better.